eng/Versions.props

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <PreReleaseVersionLabel>prerelease</PreReleaseVersionLabel>
```

Comment:
Oh, joy, we're changing the label again.  Didn't cause any problems last time (with a bystander effect of everyone assuming someone else would fix their packages :smile:).

Reply:
packages will go to a new feed with these changes, and their entire versioning label logic will change as well, so consumers will have to react to changes no matter what. @safern just suggested (offline) to use alpha instead since they are new and we might in the future want to ship beta packages that sort greater than alpha.

Reply:
Assuming the package names are staying the same (which seems legit), we should ensure that the versions are monotonic.  Which may just mean bumping to 0.1.3.

I don't think that we should jump from 0.1 to 1.0 just because we moved to arcade.

Reply:
because they are going to a different feed we might not even need the bump. We'll just tell people to reference the latest version of the packages on the new feed and that should be enough.

Reply:
> We'll just tell people to reference the latest version of the packages on the new feed and that should be enough.

I don't know if the nuget package cache considers the same name in two feeds to be different packages, or the same.  I advise assuming "same" and make sure that we don't get cache-related confusion by making sure that all new-feed packages have higher package versions than all myget-feed packages with the same name.

Reply:
It is true cache will treat them as same, but the prerelease label format  is changing as well, we used to have some special repo specific format for corefxlab packages but we are changing that to the Arcade format with this which is different and should have the packages never clash with existing ones on your cache. We can keep this in mind but the cache in theory shouldn't get in the way of figuring out which package is the latest assuming that you remove the old myget feed from your project and you now use the blob feed, even in the case where there are myget package caches on your machine.